block-layer/codecs/dag-pb.md

```diff
@@ -36,15 +36,48 @@ message PBNode {
 The objects link names are specified in the 'Name' field of the PBLink object.
 All link names in an object must either be omitted or unique within the object.
 
-## Pathing
+## Logical Format
 
-The pathing is currently different between implementations. Please see [issue #55] for more information about the harmonization effort. This section describes the current implementations as of September 2019.
+When we handle DagPB content at the Data Model level, we treat these objects as maps.
+The map keys correspond to the lowercase field names.
+
+This layout can be expressed with [IPLD Schemas](../../schemas/README.md) as:
+
+```ipldsch
+type PBLink struct {
+  hash optional Bytes
+  name optional String
+  tsize optional Int
+}
+
+type PBNode struct {
+  links [PBLink]
+  data Bytes
+}
+```
+
+The first node in a block of DagPB data will match the `PBNode` type.
+
+When decoding data with the DagPB codec, maps with exactly these fields will result.
+
+When creating data, you can create maps using the standard Data Model concepts,
+and as long as they have exactly these fields, the DagPB codec can encode them.
+If additional fields are present, the DagPB codec will error, because there is no way to encode them.
+
+## Alternative Pathing
+
+While the [logical format](#logical-format) implicitly describes set of mechanisms for pathing over and through DagPB data,
+DagPB also enjoys some other special forms of pathing in addition to the Data Model norms, and these are supported by most major applications that use DagPB.
+
+This alternative pathing is covered here as part of this descriptive spec, but was developed independently of the Data Model and is thus not well standardized.
+It currently differs between implementations. Please see [issue #55] for more information about the harmonization effort. This section describes the current implementations as of September 2019.
```

Comment:
TIL that `[issue #X]` is a thing in GFMD. But, maybe put this long-form so it's safe for publishing to specs.ipld.io when we get there. `[issue #55](https://github.com/ipld/specs/issues/55)`.

Reply:
It's one of those bottom-link things.

Reply:
oh, right, I'll hold my tongue from further comment lest it get me in trouble

Reply:
(I know -- I did a double-take when editing it too.  I'm really dubious of this feature of markdown.  If it had a different syntax -- `[[` or something -- it wouldn't be so bad; as it is, it looks to me like one just forgot to finish typing.)